docs(example): framer-motion

Anishpras opened the pull request.

kentcdodds commented on examples/with-framer-motion/app/routes/index.tsx:
```diff
@@ -0,0 +1,33 @@
+      <motion.h1 animate={{ scale: 0.5 }} transition={{ duration: 2 }}>
+        Welcome to Remix + Framer Motion
+      </motion.h1>
+      <ul>
```
Let's remove all this stuff that doesn't have anything to do with the example.

kentcdodds commented on examples/with-framer-motion/README.md:
```diff
@@ -0,0 +1,53 @@
+# Welcome to Remix + framer-motion!
```
Could you update this README to conform to the template for examples? (You can copy the template directory when creating examples to help ensure the example is simple and consistent).

kentcdodds reviewed (changes requested):
Thanks! Just a few suggestions. Also, please move this from "with-framer-motion" to simply "framer-motion". The "with-" prefix is unnecessary.

Anishpras commented:
@kentcdodds I have changed as requested. Could you please review it again? Thanks.

Anishpras commented:
> @kentcdodds I have changed as requested. Could you please review it again? Thanks.

I have changed a few more things. Thank you for your insightful reviews.

kentcdodds reviewed (approved):
Thanks!
